tests/Initialize/runner.js

```diff
@@ -0,0 +1,70 @@
+describe('Initialization.loadQml', function() {
+    it('it can load qml without a file', function() {
+        var div = loadQml("import QtQuick 2.0\nItem {}\n");
+    });
+});
+
+var QtQuickModules = [
+    {name: 'AnimatedImage', tags: []},
+    {name: 'Animation', tags: []},
+    {name: 'Behavior', tags: []},
+    {name: 'BorderImage', tags: []},
+    {name: 'Column', tags: ["dom"]},
+    {name: 'DoubleValidator', tags: []},
+    {name: 'Flow', tags: []},
+    {name: 'Font', tags: []},
+    {name: 'FontLoader', tags: []},
+    {name: 'Grid', tags: ["dom"]},
+    {name: 'Image', tags: ["dom"]},
+    {name: 'IntValidator', tags: []},
+    {name: 'ListElement', tags: []},
+    {name: 'ListModel', tags: []},
+    {name: 'ListView', tags: ["dom"]},
+    {name: 'Loader', tags: []},
+    {name: 'MouseArea', tags: ["dom"]},
+    {name: 'NumberAnimation', tags: []},
+    {name: 'ParallelAnimation', tags: []},
+    {name: 'PropertyAnimation', tags: []},
+    {name: 'Rectangle', tags: ["dom"]},
+    {name: 'RegExpValidator', tags: []},
+    {name: 'Repeater', tags: ["dom"]},
+    {name: 'Rotation', tags: []},
+    {name: 'Row', tags: ["dom"]},
+    {name: 'Scale', tags: []},
+    {name: 'SequentialAnimation', tags: []},
+    {name: 'State', tags: []},
+    {name: 'SystemPalette', tags: []},
+    {name: 'Text', tags: ["dom"]},
+    {name: 'TextInput', tags: ["dom"]},
+    {name: 'Timer', tags: []},
+    {name: 'Transition', tags: []},
+    {name: 'Translate', tags: []},
+]
```

Comment:
This does not support modules other than QtQuick. Could you add other modules support, too?
Also, this is a bit vebrose, `module.element.options` would be more compact.

Like this:
```javascript
var modules = [
  `QtQuick 2.5': {
    AnimatedImage: {},
    Animation: {},
    Behavior: {},
    BorderImage: {},
    Column: {dom: true},
    …
 }
};
```